### DISCUSSION TITLE
blinking drawXXX Elements

### DISCUSSION THREAD

**Discussion by Devilscave** [olikraus/u8g2#1668] [Q&A] [answered]
I am looking for a way to let the "drawXXX" elements flashed in time.
For example. At a Clock of issue, which is updated every second, the colons between the HH : MM in seconds Tact on/off. 
But it does not work for me. Let's get away, sometimes, just halfway or down.
This code block is within a FirstPage - NextPage drawing. The time numbers are also displayed correctly, only the points do not want to flash. Please Help

- `

        if(shine == 0){
              u8g2.drawDisk(20, 25, 2, U8G2_DRAW_ALL);
              shine++;
         }else{
              shine--;
                  }
`

>

**Comment by v1993** [comment, marked as answer]
Page loop is called multiple times during frame drawing. You want to toggle things outside of it, something like

```
// Before page loop
shine = !shine;

// Inside page loop
if (shine)
    u8g2.drawDisk(20, 25, 2, U8G2_DRAW_ALL);
```

**Reply by Devilscave** [reply]
great, now it works korrekt. Thanks  man
